recipes/qiskit-ignis/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "qiskit-ignis" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/qiskit-ignis-{{ version }}.tar.gz
+  sha256: c2351fe6c479b64c39d2786da3d0f15db122611cee039874a8979c7e583b72a5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy >=1.13
+    - python >=3.6
+    - qiskit-terra >=0.15.1
+    - retworkx >=0.8.0
+    - scipy >=0.19,!=0.19.1
+    - setuptools >=40.1.0
```

Comment:
Do you want a run constrained section?

https://github.com/Qiskit/qiskit-ignis/blob/0fb96d52060a488dbad405bb5aca49a2cda1dd1e/setup.py#L80